README.md

```diff
@@ -162,7 +163,9 @@ let edn_body = client.entity(person.crux__db___id.serialize()).unwrap();
 // )
 ```
 
-* [`entity_tx`](https://docs.rs/transistor/1.0.0-beta.3/transistor/http/struct.HttpClient.html#method.entity_tx) requests endpoint [`/entity-tx`](https://opencrux.com/docs#rest-entity-tx) via `POST`. A serialized `CruxId`, serialized `Edn::Key` or a String containing a [`keyword`](https://github.com/edn-format/edn#keywords) must be passed as argument. Returns the transaction that most recently set a key.
+* [`entity_timed`](https://docs.rs/transistor/1.0.0/transistor/http/struct.HttpClient.html#method.entity_timed) is similar to `entity` as it requests the same endpoint, the difference is that it can send `transaction-time` and `valid-time` as query-params. This is done by the extra arguments `transaction_time: Option<DateTime<FixedOffset>>` and `valid_time: Option<DateTime<FixedOffset>>`.
+
+* [`entity_tx`](https://docs.rs/transistor/1.0.0/transistor/http/struct.HttpClient.html#method.entity_tx) requests endpoint [`/entity-tx`](https://opencrux.com/docs#rest-entity-tx) via `POST`. A serialized `CruxId`, serialized `Edn::Key` or a String containing a [`keyword`](https://github.com/edn-format/edn#keywords) must be passed as argument. Returns the transaction that most recently set a key.
```

Comment:
Shouldn't this be before the one with time? Since the other one is referencing this one

Reply:
github made this weird, but they are correctly grouped. I wouldn't reference something I have not said yet, also, the other one should come first as it has the example.

Reply:
Are you sure? Both preview and code seem like the first one is `entity_timed` and then `entity_tx`
![Screenshot from 2020-08-10 05-43-09](https://user-images.githubusercontent.com/15306309/89765373-90752000-dacc-11ea-8364-c10f5e0cd614.png)

Reply:
first is entity then entity_timed, then entity_tx then entity_tx_timed.
```
* [`entity`](https://docs.rs/transistor/1.0.0/transistor/http/struct.HttpClient.html#method.entity) requests endpoint [`/entity`](https://opencrux.com/docs#rest-entity) via `POST`. A serialized `CruxId`, serialized `Edn::Key` or a String containing a [`keyword`](https://github.com/edn-format/edn#keywords) must be passed as argument. Returns an entity for a given ID and optional valid-time/transaction-time co-ordinates.
...

* [`entity_timed`](https://docs.rs/transistor/1.0.0/transistor/http/struct.HttpClient.html#method.entity_timed) is similar to `entity` as it requests the same endpoint, the difference is that it can send `transaction-time` and `valid-time` as query-params. This is done by the extra arguments `transaction_time: Option<DateTime<FixedOffset>>` and `valid_time: Option<DateTime<FixedOffset>>`.

* [`entity_tx`](https://docs.rs/transistor/1.0.0/transistor/http/struct.HttpClient.html#method.entity_tx) requests endpoint [`/entity-tx`](https://opencrux.com/docs#rest-entity-tx) via `POST`. A serialized `CruxId`, serialized `Edn::Key` or a String containing a [`keyword`](https://github.com/edn-format/edn#keywords) must be passed as argument. Returns the transaction that most recently set a key.
...
* [`entity_tx_timed`](https://docs.rs/transistor/1.0.0/transistor/http/struct.HttpClient.html#method.entity_tx_timed) is similar to `entity_tx` as it requests the same endpoint, the difference is that it can send `transaction-time` and `valid-time` as query-params. This is done by the extra arguments `transaction_time: Option<DateTime<FixedOffset>>` and `valid_time: Option<DateTime<FixedOffset>>`.
```